resources/README.md

```diff
@@ -0,0 +1,117 @@
+# Stolon Helm Chart
+
+* Installs [Stolon](https://github.com/sorintlab/stolon) (HA PostgreSQL cluster)
+* Based on [lwolf/stolon-chart](https://github.com/lwolf/stolon-chart) and [stolon examples](https://github.com/sorintlab/stolon/tree/master/examples/kubernetes/statefulset)
+
+
+Official upstream `helm/charts` support was deprecated on Nov 13, 2020. This a
+community effor to maintain the helm chart. Please create an issue if you're
+interested on helping to maintain this chart.
+
+
+## TL;DR;
+
+```console
+$ helm repo add iomedstolon https://iomedhealth.github.io/helm_stolon/
+$ helm install iomedstolon/stolon
+```
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```console
+$ helm install --name my-release iomedstolon/stolon
+```
+
+## Backend
+
+Kubernetes is the default store backend. `consul`, `etcdv2` or `etcdv3` can also be used as the store backend.
```

Comment:
Kubernetes should be `"kubernetes"` e.g. in quotes since it should be a backend name?